talk/owt/sdk/include/cpp/owt/base/globalconfiguration.h

```diff
@@ -83,6 +83,16 @@ class GlobalConfiguration {
           audio_frame_generator_.reset(nullptr);
       }
   }
+  /**
+   @brief This function sets the temporal layers for h264.
+   @param num_temporal_layers Number of temporal layers for h.264. Value greater
+   than 4 will be ignored.
+  */
+  static void SetH264EncoderTemporalLayers(int layers) {
```

Comment:
Temporal layer settings are expected to be configured through encoding parameters.

Reply:
That's true. The code for configuring via Encoding parameters is also included in this patch. However, upstream does not really yet pass it to video engine, 
Also this is not in spec so setting via encoding parameter is not always guarenteed to be kept.

Reply:
It's defined in [Scalable Video Coding (SVC) Extension for WebRTC](https://www.w3.org/TR/webrtc-svc/), not webrtc-pc.

Please describe the reason in code, and remove this API when encoding parameters take effect in the future. They may conflict with each other.

Reply:
done.